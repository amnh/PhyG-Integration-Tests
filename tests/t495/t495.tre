(AlbanianTosk:370.45,(Yiddish:4513.299999999999,(Danish:1450.1999999999998,(Irish:1739.5,(Icelandic:8495.9,(ArmenianWestern:3288.05,((Scots:2245.3,(((EasternFrisianSaterland:1642.95,(AncientIonicGreek:640.25)#HTU36:640.25)HTU17:771.6,(Manx:1806.45,#HTU32:159.5)HTU31:1006.6)HTU28:1345.6,#HTU34:988.0)HTU33:1397.3)HTU29:1899.7,(Cornish:1752.5,((KolschColognian:7566.75,(OldPrussian:1648.1,(Faroese:340.0)#HTU32:340.0)HTU22:673.55)HTU20:834.1,((SanskritVedic:1334.1,(Assamese:1208.6,#HTU36:501.5)HTU35:1179.3000000000002)HTU21:19.1)#HTU34:19.1)HTU19:518.65)HTU30:2167.6)HTU18:1529.1)HTU23:2132.1)HTU25:2582.45)HTU26:956.6500000000001)HTU24:1067.45)HTU27:767.95)HTU16[45291.6];
