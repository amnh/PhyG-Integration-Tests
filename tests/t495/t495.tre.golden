(AlbanianTosk:0.0,(((Irish:274.9,(Cornish:185.0,(Manx:168.5,(Faroese:325.2,(SanskritVedic:158.2,(Yiddish:554.0,#HTU34:0.0)HTU33:40.0)HTU22:324.0)HTU23:183.6)HTU20:39.1)HTU24:137.0)HTU21:85.2,((((Icelandic:483.3,KolschColognian:434.0)HTU25:247.6,#HTU36:0.0)HTU35:12.0,(Assamese:0.0)#HTU34:0.0)HTU26:15.9,#HTU32:0.0)HTU31:161.5)HTU29:94.0,((ArmenianWestern:353.2,(OldPrussian:246.3,((AncientIonicGreek:651.3,Danish:237.9)HTU17:0.0)#HTU36:0.0)HTU18:156.0)HTU28:65.4,((EasternFrisianSaterland:291.9,Scots:274.4)HTU27:0.0)#HTU32:0.0)HTU30:6.0)HTU19:0.0)HTU16[11091.5];
